eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -1857,11 +2164,13 @@ groups:
           central: 1.7
           min:     1.7
           max:     1.7
+          latex:    '$s_0^{B \to K^*, \text{(LCSR)}}$'
           comments: "GeV^2"
       "B->K^*::M^2@LCSR" :
           central: 1.0
           min:     0.75
           max:     1.25
+          latex:    '$(M^2)^{B \to K^*, \text{(LCSR)}}$'
```

Comment:
Why do you have parenthesis around `M^2`?

Reply:
I liked it better.
It somewhat suggests that it's M^2 with the name outside the paranthesis. Buuuut... yeah, just got aware it's the Borel parameter and its written differently for DKMMO2008

Reply:
Ok fine, I wasn't looking at the double superscript